### PR TITLE
dotnet example (RK-5747)

### PR DESCRIPTION
**Pull request by BenTheKlein**

### CODE REVIEW

**Comment by BenTheKlein** [comment]
@yonitwis @ElDuderinos i will really appreciate it if you could run it and say what you think

File: java-gradle/java-api/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Before following this guide we recommend reading the basic [Java + Rookout] guid
     git clone https://github.com/Rookout/deployment-examples.git
     cd deployment-examples/java-gradle/java-api
     ```
-2. Add your otganization token to line 14 at HelloWorld.java
+2. Add your organization token to line 14 at HelloWorld.java
```
Comment (LiranLast) [inline java-gradle/java-api/README.md:16]: Maybe use an env variable instead?

File: dotnet-aspnet/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Before following this guide we recommend reading the basic [DotNet + Rookout] guide.
+
+## Running local application
+
+1. Clone and compile the project:
+     ```bash
+    git clone https://github.com/Rookout/deployment-examples.git
+    cd deployment-examples/dotnet-aspnet/dotnet-core-3
+    ```
+2. Add your organization token to line 21 at Program.cs
```
Comment (LiranLast) [inline dotnet-aspnet/README.md:14]: Maybe use env variable?

File: java-gradle/java-api/README.md (hunk 1 above)
Comment (BenTheKlein) [reply java-gradle/java-api/README.md:16]: ok

**Comment by BenTheKlein** [comment]
@LiranLast @yonitwis mind approving this?

**Comment by BenTheKlein** [comment]
@yonitwis This is pending your approval

**Pull request merged**